app/config/site-templates.php

```diff
@@ -84,7 +84,7 @@ function getFramework(string $frameworkEnum, array $overrides)
 return [
     [
         'key' => 'nextjs-starter',
-        'name' => 'Next.js Starter website',
+        'name' => 'Next.Js starter website',
```

Comment:
```suggestion
        'name' => 'Next.js starter website',
```
![CleanShot 2024-11-26 at 15 28 19@2x](https://github.com/user-attachments/assets/ec29070d-a1af-484b-80ad-75d4d5688e2a)